yarn.lock

```diff
@@ -12053,6 +12058,14 @@ dezalgo@^1.0.0:
     asap "^2.0.0"
     wrappy "1"
 
+dezalgo@^1.0.4:
```

Comment:
Please consolidate this with the `dezalgo` dependency above so they both use the latest version